backend/controllers/getListingController.js

```diff
@@ -0,0 +1,39 @@
+const Listing = require('../models/listingModel');
+const Homeowner = require('../models/homeownerModel');
+
+// function to view stand-alone listing [common to HO and CO]
+const getListing= async (req, res) => {
+    try {
+      // Find the listing by ID
+      const listing = await Listing.findById(req.params.listing_id).select('-homeowner_id');
```

Comment:
Is the hyphen required here?

Reply:
Yes. This is excluding the homeowner_id to be returned in the response body to the user.

Reply:
I see, this makes sense.